x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -193,6 +200,10 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
 
   const setExceptionItemsToAdd = useCallback(
     (items: ExceptionsBuilderReturnExceptionItem[]): void => {
+      dispatch({
+        type: 'setWildcardWithWrongOperator',
+        warningExists: getHasWrongOperator(items),
```

Comment:
Consider renaming constants/fields/functions to one (or at least similar name) - now there's too many variations for warnings

Reply:
Talked with @tomsonpl  offline about the need for descriptive names since we have several different functions dealing with wildcards that all do different things.